internal/pkg/cli/app_show.go

```diff
@@ -121,7 +132,45 @@ func (o *showAppOpts) description() (*describe.App, error) {
 	if err != nil {
 		return nil, fmt.Errorf("list jobs in application %s: %w", o.name, err)
 	}
+	var mux sync.Mutex
+	g, ctx := errgroup.WithContext(context.Background())
+	defer ctx.Done()
+	for i := range envs {
+		env := envs[i]
```

Comment:
```suggestion
	for _, env := range envs {
```

Reply:
this won't actually work (😢) - https://go.dev/doc/faq#closures_and_goroutines

Reply:
oooh that's right!